.github/workflows/aarch64-sysroot.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: aarch64 sysroot
+on:
+  pull_request:
+    paths:
+      - .github/workflows/aarch64-sysroot.yaml
+  push:
+    tags:
+      - "aarch-sysroot-*"
+
+jobs:
+  toolchains:
+    runs-on: ubuntu-22.04
+    name: aarch64 sysroot
+    steps:
+      - name: Download & adjust script
+        run: |
+          wget -c https://chromium.googlesource.com/chromium/src/+archive/main/build/linux/sysroot_scripts.tar.gz -O - | tar -xz
```

Comment:
I think we might want to vendor these scripts (check them into the repo), rather than download them from the internet every build.